explore/src/main/scala/explore/constraints/ConstraintsPanel.scala

```diff
@@ -194,76 +191,49 @@ object ConstraintsPanel {
         } yield ()
 
       val editWarning =
-        state.get.showMultiEditWarning
-          .whenDefined(shown =>
-            Segment(inverted = true,
-                    color = Yellow,
-                    textAlign = textalignment.Center,
-                    size = Small,
-                    clazz = ExploreStyles.EditWarning
-            )(
-              // If we actually render this when shown, it will trigger a rerender, which will remove focus from selected input.
-              ^.hidden := !shown
-            )(
-              s"These constraints are shared by $obsCount observations and editing will modify all of them.",
-              <.br,
-              "To modify only this observation, you can ",
-              Button(compact = true,
-                     size = Small,
-                     basic = true,
-                     circular = true,
-                     inverted = true,
-                     clazz = ExploreStyles.ButtonCopy,
-                     onClick = makeCopy.runAsyncCB
-              )(
-                "make a copy"
+        <.span(
+          Popup(
+            trigger =
+              Label(as = <.a, circular = true, color = colors.Red)("Shared ",
+                                                                   Icons.Question.fitted(true)
               ),
-              ".",
-              Button(
-                compact = true,
-                size = Small,
-                basic = true,
-                circular = true,
-                inverted = true,
-                floated = floats.Right,
-                clazz = ExploreStyles.ButtonDismiss,
-                onClick = state.zoom(State.showMultiEditWarning).set(none).runAsyncCB
-              )("Dismiss")
-            )
+            content = PopupContent(
+              content = <.div(ExploreStyles.SharedEditWarning)(
+                s"These constraints cannot be modified because they are shared by $obsCount observations.",
+                <.br,
+                "Please edit in Constraints View, create a New one or a Copy."
+              )
+            ),
+            wide = PopupWide.Very,
+            position = PopupPosition.TopCenter
           )
-          .when(props.multiEditWarnings)
+        ).when(!props.allowMultiEdit)
```

Comment:
`.unless(props....)`? It is just taste feel free to ignore

Reply:
Good catch! Will change.